docs/source/gallery-jhub-deployments.md

```diff
@@ -160,6 +160,10 @@ easy to do with RStudio too.
 - https://getcarina.com/blog/learning-how-to-whale/
 - http://carolynvanslyck.com/talk/carina/jupyterhub/#/
 
+### Hadoop
+
+- [Deploying JupyterHub on Hadoop](https://jcrist.github.io/jupyterhub-on-hadoop/)
```

Comment:
This is written in the style of `zero-to-jupyterhub-k8s` but for Hadoop clusters. As it's seen less testing than TLJH and zero-to-jupyterhub I figured it'd be best to put it in the gallery instead of in the distributions list. Not sure what's best.

Reply:
Makes sense to me. If you feel it would be better moved over later, feel free to come on back.